core/src/wsv.rs

```diff
@@ -386,6 +386,7 @@ impl WorldStateView {
     ///
     /// # Errors
     /// - There is no account with such name.
+    // NOTE: Allowed because of assert!
```

Comment:
```suggestion
    /// # Panics
    /// - if [`Account::add_asset`] fails.   
```

If you're documenting allowing an undocumented panic, you should just document the panic.